### PR TITLE
Trying NDEBUG.

### PR DESCRIPTION
**Pull request by dkorolev**
This passes Travis tests.

### CODE REVIEW

File: port.h [hunk 1]
```diff
@@ -217,4 +217,10 @@ inline const char* strptime(const char* input_string, const char* input_format,
 using namespace enable_strptime_on_windows;
 #endif  // CURRENT_WINDOWS
 
+inline void CURRENT_ASSERTION_FAILED(const char* text, const char* file, int line) {
+  fprintf(stderr, "Current assertion failed:\n\t%s\n\t%s : %d\n", text, file, line);
+}
+
+#define CURRENT_ASSERT(expr) ((expr) ? static_cast<void>(0) : CURRENT_ASSERTION_FAILED(#expr, __FILE__, __LINE__))
```
Comment (mzhurovich) [inline port.h]: `#ifndef CURRENT_ASSERT`?

File: Bricks/net/http/impl/server.h [hunk 2]
```diff
@@ -218,7 +218,8 @@ class GenericHTTPRequestData : public HELPER {
                 //    during the next iteration of the outer loop.
                 // The original version of this code was only adding one to `next_offset`.
                 // This had a bug, which got revealed as the `while` loop above has been corrected into `if`.
-                // Upon changing the `while` to an `if`, the `assert (buffer_.size() > offset + 1);` check above
+                // Upon changing the `while` to an `if`, the `CURRENT_ASSERT(buffer_.size() > offset + 1);`
+                // check above
```
Comment (mzhurovich) [inline Bricks/net/http/impl/server.h]: indent :(

File: Bricks/net/http/headers/headers.h [hunk 3]
```diff
@@ -253,7 +254,8 @@ struct Headers final {
     Header::ThrowIfHeaderIsCookie(header);
     const auto rhs = map.find(header);
     if (rhs != map.end()) {
-      assert(rhs->second);  // Invariant: If `header` exists in map, its `std::unique_ptr<>` value is valid.
+      CURRENT_ASSERT(
+          rhs->second);  // Invariant: If `header` exists in map, its `std::unique_ptr<>` value is valid.
```
Comment (mzhurovich) [inline Bricks/net/http/headers/headers.h]: indent :(

File: Bricks/net/http/headers/headers.h [hunk 4]
```diff
@@ -239,7 +239,8 @@ struct Headers final {
...
```
Comment (mzhurovich) [inline Bricks/net/http/headers/headers.h]: indent :(

File: Bricks/net/http/headers/headers.h [hunk 5]
```diff
@@ -228,7 +227,8 @@ struct Headers final {
...
```
Comment (mzhurovich) [inline Bricks/net/http/headers/headers.h]: indent :(

File: port.h (hunk 1 above)
Comment (dkorolev) [inline port.h]: :+1: 9edc455

File: Bricks/net/http/headers/headers.h (hunk 3 above)
Comment (dkorolev) [inline Bricks/net/http/headers/headers.h]: Fixed indent issues in 09c4163.

File: Bricks/net/http/impl/server.h (hunk 2 above)
Comment (dkorolev) [inline Bricks/net/http/impl/server.h]: This one too, in 09c4163.

**Pull request merged**